src/components/Theme/constants.scss

```
@@ -73,6 +73,7 @@ $color-navbar-gradient-two: rgba(28, 31, 33, 0.35);
 $color-small-button: rgba(15, 37, 54, 0.85);
 $color-small-button-hover: rgba(15, 37, 54, 0.6);
 $color-small-button-active: rgba(15, 37, 54, 0.85);
+$color-icon-button-hover: #d5dae0;
```

I think we should avoid this name for this new button component's color constant as there are existing customisation options, tokens for an unrelated/different Icon Buttons UI.

---

I'm about to move it to `tokens.scss` and name it as:

```suggestion
  --osdk-color-background-button-custom: #d5dae0;
```

What do you think?

---

Great! I'd just drop the `osdk-` prefix for non-customisable UI tokens.